doc: add supported architectures for the agent (#35)
rancher/stackstate-product-docs: docs/next/modules/en/pages/setup/agent/supported-architectures.adoc

= SUSE Observability Agent supported architectures

> Please note that SUSE Observability is designed and tested to run on specific hardware architectures. Installation and operation are exclusively supported on systems utilizing one of the following processor architectures:
> 
> * `x86_64` (also known as AMD64 or Intel 64)
> * `arm64` (also known as AArch64)
> 
> Ensure your target environment meets these requirements before proceeding with the deployment. Attempting to install or run SUSE Observability on any other architecture is not supported and will result in failure.

Reviewer: A little shorter.
```suggestion
SUSE Observability is designed and tested to run on specific hardware architectures. Installation and operation are supported on these systems:

* `x86_64` (also known as AMD64 or Intel 64)
* `arm64` (also known as AArch64)

Ensure your target environment meets these requirements before proceeding with a deployment. Installing or running SUSE Observability on other architectures is not supported.
```
